scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/opto.yaml

```diff
@@ -0,0 +1,29 @@
+Broadcom_DFN-6_2.0x2.0mm:
+  size_source: 'https://docs.broadcom.com/docs/AV02-4755EN'
+  library: 'OptoDevice'
+  manufacturer: 'Broadcom'
+  device_type: 'DFN'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  body_size_x:
```

Comment:
Capture the info just as it is in the datasheet. So here, instead of min/nom/max, you would put:
```
body_size_x:
    nominal: 2
    tolerance: 0.1
```